src/mandelbrot.rs

```diff
@@ -3,15 +3,13 @@
 //
 // contributed by Matt Watson
 // contributed by TeXitoi
+// contributed by Cristi Cobzarenco (@cristicbz)
```

Comment:
feel free to let it like that if you want to submit the bench yourself, or let my name at the end if you want I take care of the submission.